### PR TITLE
Errors for Patron Codes

### PR DESCRIPTION
**Pull request by heymatthew**

### CODE REVIEW

File: src/api/patron-codes.md [hunk 1]
```diff
@@ -125,4 +125,10 @@ Request][polling] using the transacting APIs.
 }
 ```
 
+{% h4 Error Responses %}
+
+| Status |          Code              |                   Description                |
+| :----- | :--------------------------| :--------------------------------------------|
+| 403    | {% break _ INVALID_LUHN %} | Luhn checksum digit doesn't pass, try again. |
```
Comment (ncjones) [inline src/api/patron-codes.md:132]: can we say "CHECKSUM_FAILED" and remove "try again"?

Comment (heymatthew) [reply src/api/patron-codes.md]: done :)

File: src/api/payment-requests/legacy-payment-requests.md [hunk 2]
```diff
@@ -65,6 +65,13 @@ Legacy Payment Request endpoints also have
 | deviceId             | Physical payment system device Id                                                  |
 | patronCode           | Associate this payment request with an active Patron Code. |
 
+{% h4 Error Responses %}
+
+| Status |          Code                     |             Description                              |
+| :----- | :---------------------------------| :----------------------------------------------------|
+| 403    | {% break _ CHECKSUM_FAILED %}     | patronCode luhn checksum digit doesn't pass.         |
+| 403    | {% break _ PATRON_CODE_EXPIRED %} | patronCode has expired so cannot be used, try again. |
```
Comment (ncjones) [inline src/api/payment-requests/legacy-payment-requests.md:73]: stray "try again"

File: src/api/patron-codes.md [hunk 3]
```diff
@@ -125,4 +125,10 @@ Request][polling] using the transacting APIs.
 }
 ```
 
+{% h4 Error Responses %}
+
+| Status |          Code                 |          Description              |
+| :----- | :-----------------------------| :---------------------------------|
+| 403    | {% break _ CHECKSUM_FAILED %} | Luhn checksum digit doesn't pass. |
```
Comment (ncjones) [inline src/api/patron-codes.md:132]: actually, should these be 400 bad request?

File: src/api/payment-requests/legacy-payment-requests.md (hunk 2 above)
Comment (heymatthew) [reply src/api/payment-requests/legacy-payment-requests.md]: Removed this text.

File: src/api/patron-codes.md (hunk 3 above)
Comment (heymatthew) [reply src/api/patron-codes.md]: Gone with 400 for checksum, and 410 Gone for expired.

**Review by ncjones — APPROVED** [review]

**Pull request merged**